tests/queries/0_stateless/01779_quantile_sanitizer.sql

```diff
@@ -0,0 +1,3 @@
+SELECT cityHash64(toString(quantileDeterministicState(number, sipHash64(number)))) FROM numbers(8193);
```

Comment:
Ok, though I prefer adding a new test for every test case - it allows more quickly figure out the reason when test will fail.